config/routes.rb

```diff
@@ -16,5 +16,11 @@
     end
   end
 
+  resources :reviews do
+    member do
+      get :delete
```

Comment:
if you remove all the `erb` files and just go with jsons you can remove this action.